.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```

Comment:
This file is nice to have, so we don't have eslint and the code editor fighting over the style.